cmd/sourced/cmd/web.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/src-d/sourced-ce/cmd/sourced/compose"
 )
 
+// The service name used in docker-compose.yml for the srcd/sourced-ui image
+const containerName = "sourced-ui"
```

Comment:
maybe it should be env variable? just a suggestion. I'm good with hard code.

Reply:
:thinking: If we consider you can download any docker-compose.yml file, I guess in theory the service name could change. But for CE and EE I don't think we it, probably it's a premature optimization to add this level of customization.